main.go

```diff
@@ -171,18 +171,36 @@ func registerDynamicCapabilities(port string, log logger.Handler) {
 
 }
 func registerWorkloads(port string, log logger.Handler) {
-	release, err := config.GetLatestReleases(1)
-	if err != nil {
-		log.Info("Could not get latest stable release")
-		return
+	var url string
+	var gm string
+	//If a URL is passed from env variable, it will be used for component generation with default method being "using manifests"
+	// In case a helm chart url is passed,COMP_GEN_METHOD env variable should be set to Helm otherwise the component generation fails
```

Comment:
```suggestion
	// In case a helm chart URL is passed, COMP_GEN_METHOD env variable should be set to Helm otherwise the component generation fails
```